test.md

```diff
@@ -0,0 +1 @@
+# This file should be ignored by git
```

Comment:
- [ ] checking the project [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), kindly add this file to your git ignore file

Reply:
Thanks a lot for your kind review!